types/src/working-group/index.ts

```diff
@@ -300,6 +300,15 @@ export class RewardPolicy extends JoyStruct<IRewardPolicy> {
       payout_interval: 'Option<BlockNumber>',
     }, value);
   }
+  get amount_per_payout(): Balance {
+    return this.getField<Balance>('amount_per_payout');
```

Comment:
is this syntax better than `this.getField('amount_per_payout') as Balance;` ?
I've seen eslint in other project recommend it.

Reply:
`return this.getField('amount_per_payout') as Balance;`
would equal to:
`return <Balance> this.getField('amount_per_payout');`
but not:
`this.getField<Balance>('amount_per_payout');`

The last one is something else, because this way we actually "pass" the `Balance` as a generic-type "argument" value to `getField`.

The definition of `getField` is as follows:
```
  getField<C extends Codec> (name: keyof T): C {
    return super.get(name as string) as C;
  }
```

This means, that in that case it's aditionally enforced that `Balance` extends `Codec` (which wouldn't be the case with `getField(...) as Balance`). This is definitely not an anti-pattern.

Reply:
That makes sense, then I think there are quite a few places we need to correct the getters of the JoyStruct.